rts/motoko-rts/src/gc/mark_compact.rs

```diff
@@ -18,7 +18,11 @@ use motoko_rts_macros::ic_mem_fn;
 
 #[ic_mem_fn(ic_only)]
 unsafe fn schedule_compacting_gc<M: Memory>(mem: &mut M) {
-    if super::should_do_gc() {
+    // Mark stack ignored. Max. bitmap can be 130,150,524 bytes.
+    // (x + x / 32 = 4 GiB, x = 4,164,816,771, x/32 = 130,150,524)
+    const MAX_LIVE: Bytes<u64> = Bytes(4_164_816_771);
```

Comment:
Please express this in terms of max-wasm-memory-size and max-marking-bitmap-size.

Shall we also add a some slack for subsequent message execution and marking stack? The copying GC doesn't have this problem because the mutator can continue allocating in the second half of the heap.

To be on the safe side, let's leave 500MB-1GB of slack? 

`MAX_LIVE = MAX_WASM_MEMORY - MAX_MARKING_BITMAP - SLACK`

Reply:
Hmm, maybe we could put all but the first page of markstack in stable memory in future, paging out/in in stack order....

Reply:
I reserved 500 MiB for the mark stack.